src/components/Dashboard/TodoList.vue

```diff
@@ -859,7 +839,7 @@ export default class TodoList extends Vue {
       const corpTypeDescription = this.getCorpTypeDescription(business.legalType)
 
       const paymentStatusCode = header.paymentStatusCode
-      const payErrorObj = paymentStatusCode && await PayServices.getPayErrorObj(paymentStatusCode)
+      const payErrorObj = paymentStatusCode && await PayServices.getPayErrorObj(paymentStatusCode, this.getPayApiUrl)
```

Comment:
How about standardizing on formal parameter order: URL first, then codes/ids/etc (same between all services?)

Reply:
^^ I wrote this before checking the other services calls. Maybe no need to change...

Reply:
Yes, I agree, normally I'd place the URL first, but in this case, I placed it second so that it could be optional.  
```
static async getPayErrorObj (code: string, payApiUrl: string = ''): Promise<PaymentErrorIF> {
    const url = `${payApiUrl}codes/errors/${code}`
    return axios.get(url)
      .then(response => response?.data)
      .catch(() => {}) // ignore errors
  }
```

Let me know if you think I should make the url parameter required and, thereby, allow the parameter order to be reversed.

Reply:
Is it ever not required?

Reply:
Fair point!  I had made it optional because I thought that I wouldn't have to change a series of tests that call the function, but I now see there's only one test.  I'll change it.